### PR TITLE
blockcopy: Extend case to copy disk to NFS and local disk

### PR DESCRIPTION
**Pull request by will-Do**
Signed-off-by: Yanbing Du <ydu@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_blockcopy.py [hunk 1]
```diff
@@ -251,6 +243,9 @@ def run(test, params, env):
         if params.get('setup_libvirt_polkit') == 'yes':
             raise error.TestNAError("API acl test not supported in current"
                                     + " libvirt version.")
+        if not copy_to_nfs:
+            raise error.TestNAError("Bug will not fix:"
```
Comment (yumingfei) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blockcopy.py]: Does it mean non-copy_to_nfs is not implemented yet?
So why should we add testcases about it here?

Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blockcopy.py]: The bug will not fix on RHEL6, but fixed on RHEL7.
This 'if' block is under: 
```if not libvirt_version.version_compare(1, 1, 1):```

Comment (yumingfei) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blockcopy.py]: Got it. thanks~

**Comment by yumingfei** [comment]
Have you got the result?

**Comment by will-Do** [comment]
Well, just run the test in my box:
```
# ./run -t libvirt --test type_specific.ydu-libvirt.virsh.blockcopy..nfs_disk.. -k
Running setup. Please wait...
SETUP: PASS (3.38 s)
DATA DIR: /home/ydu/Work/virt-test/shared/data
DEBUG LOG: /home/ydu/Work/virt-test/logs/run-2015-01-09-12.06.57/debug.log
TESTS: 22
(1/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.no_option: PASS (42.30 s)
(2/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.reuse_external: PASS (52.01 s)
(3/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.raw_format: PASS (37.95 s)
(4/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.min_bandwidth.normal: PASS (694.70 s)
(5/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.max_bandwidth: PASS (40.09 s)
(6/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.shallow_option: PASS (39.66 s)
(7/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.wait_option: PASS (41.35 s)
(8/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.pivot_option: PASS (39.72 s)
(9/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.finish_option: PASS (38.73 s)
(10/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.pivot_async_option: PASS (42.29 s)
(11/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_local.finish_async_option: PASS (42.77 s)
(12/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.no_option: PASS (46.99 s)
(13/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.reuse_external: PASS (65.49 s)
(14/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.raw_format: PASS (42.93 s)
(15/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.min_bandwidth.normal: PASS (698.63 s)
(16/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.max_bandwidth: PASS (46.76 s)
(17/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.shallow_option: PASS (46.51 s)
(18/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.wait_option: PASS (46.64 s)
(19/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.pivot_option: PASS (49.07 s)
(20/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.finish_option: PASS (46.60 s)
(21/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.pivot_async_option: PASS (44.57 s)
(22/22) type_specific.ydu-libvirt.virsh.blockcopy.positive_test.non_acl.nfs_disk.dest_nfs.finish_async_option: PASS (45.01 s)
TOTAL TIME: 2292.21 s (38:12)
TESTS PASSED: 22
TESTS FAILED: 0
SUCCESS RATE: 100.00 %

**Comment by yumingfei** [comment]
OK.

**Pull request merged**